.buildkite/pipelines/code_coverage/daily.yml

```diff
@@ -0,0 +1,41 @@
+env:
+  SLACK_NOTIFICATIONS_CHANNEL: "#kibana-qa"
+  SLACK_NOTIFICATIONS_ENABLED: "true"
+  CODE_COVERAGE: 1
+notify:
+  - slack:
+      channels:
+        - "#kibana-qa"
+steps:
+  - command: .buildkite/scripts/lifecycle/pre_build.sh
+    label: Pre-Build
+    timeout_in_minutes: 10
+    agents:
+      queue: kibana-default
+
+  - wait
+
+  - command: .buildkite/scripts/steps/test/pick_test_group_run_order.sh
+    label: 'Pick Test Group Run Order'
+    agents:
+      queue: kibana-default
+    env:
+      FTR_CONFIGS_DEPS: ''
+      LIMIT_CONFIG_TYPE: 'unit,functional,integration'
+      JEST_UNIT_SCRIPT: '.buildkite/scripts/steps/code_coverage/jest.sh'
+      JEST_INTEGRATION_SCRIPT: '.buildkite/scripts/steps/code_coverage/jest_integration.sh'
+      FTR_CONFIGS_SCRIPT: '.buildkite/scripts/steps/code_coverage/ftr_configs.sh'
+
+  - command: .buildkite/scripts/steps/code_coverage/ingest.sh
+    label: 'Merge and Ingest'
+    agents:
+      queue: c2-16
+    depends_on:
+      - jest
+      - jest-integration
+      - ftr-configs
+    timeout_in_minutes: 30
+    key: ingest
+
+  - wait: ~
```

Comment:
Pretty sure this step isn't doing anything

Reply:
I'll drop it